web-app-2/mysql/src/main/java/com/m0ncld/sso/webapp2/util/GenericDbRepository.java

```diff
@@ -0,0 +1,167 @@
+package com.m0ncld.sso.webapp2.util;
+
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+
+/**
+ * Generic database repository
+ * @param <M> Model
+ * @param <I> Model identifier
+ * @param <E> Entity
+ * @param <R> Spring database CRUD repository
+ */
+public class GenericDbRepository<M extends Model<I>, I, E extends Model<I>, R extends CrudRepository<E, I>> implements GenericCrudRepository<M, I> {
```

Comment:
`M extends Model<I>` oraz  `E extends Model<I>` wskazuje, że konwencja jest taka, że Model to klasa zarówno dla obiektów domenowych, jak i encji - nie przywalam się, bo dziś zauważyłem podobną konwencję u mnie w robocie i widzę, że model jest określeniem dość wszechstronnym